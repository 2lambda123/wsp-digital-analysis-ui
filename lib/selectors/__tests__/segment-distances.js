/* global describe, it, expect */
import selectSegmentDistances from '../segment-distances'

describe('Selectors > Segment Distances', () => {
  it('should get the distances correctly for the active modifications segments', () => {
    expect(selectSegmentDistances({
      scenario: {
        modifications: [{
          id: '1',
          segments: [{
            geometry: {
              type: 'LineString',
              coordinates: [
                [-77, 38],
                [-77, 39]
              ]
            }
          }]
        }]
      }
    }, {
      params: {
        modificationId: '1'
      }
    }).map(v => v.toFixed(8))).toMatchSnapshot()
  })
})
