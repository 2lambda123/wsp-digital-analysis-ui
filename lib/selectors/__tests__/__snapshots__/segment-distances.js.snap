// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Selectors > Segment Distances should get the distances correctly for the active modifications segments 1`] = `
Array [
  "111.22983323",
]
`;
